css/style.css

```diff
@@ -1,35 +1,79 @@
+
+body{
+padding:0px;
+margin:0px;
+}
+#main{
+	width: 100%;
+    min-height: 650px;
+	background-image: url("img-main-backgroung.gif");
+	background-size: cover;
+	-webkit-background-size: cover;
+
+}
 #canvas {
   width: 566px;
-  height: 575px;
-  border: 1px solid black;
-  background-image: url("building.gif");
+  height: 566px;
+  background-image: url("building2.gif");
   display: none;
+  margin-left: auto;
+  margin-right: auto;
 }
 #menu {
-	width:90%;
-	height:570px;
-	background-image: url("building.gif");
+	width: 100%;
+    min-height: 650px;
+	background-image: url("img-menu-background.gif");
+	background-size: cover;
+	margin-left: auto;
+  	margin-right: auto;
 	background-repeat: no-repeat;
-/*	position: absolute;
-*/}
-#start {
+}
+.img-menu-girl{
 	position: relative;
-	top: 200px;
+	display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#start {
+	position: absolute;
+	top: 397px;
 	padding: 15px;
-	margin:auto;
 	display: block;
+	background:url(../design/Menu/play-button.gif);
```

Comment:
please, avoid an upper-case chars in links. There are a differences between *nix & windows os for that case.